stackslib/src/chainstate/burn/db/sortdb.rs

```diff
@@ -4852,6 +4902,16 @@ impl SortitionDB {
             None => Ok(None),
         }
     }
+    
+    /// Get a block snapshot for a winning Nakamoto tenure in a given burn chain fork.
```

Comment:
What does the function really do -- from this description it isn't clear to me. It seems like its querying by the committed-to block id? Maybe

```
/// Get the block snapshot of a tenure in the burn chain fork specified by `tip` where the winning block commit specified `last_tenure_id` as the most recent stacks block.
```